src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -411,8 +411,7 @@ def default_api_version(self):
         'vm_default_target_network': '2018-01-01',
         'nw_connection_monitor': '2019-06-01',
         'container_network': '2018-08-01',
-        'appservice_network': '2020-04-01',
-        'appservice_ensure_subnet': '2019-02-01'
+        'appservice_network': '2021-08-01'
```

Comment:
I think it's better for us to totally remove adhoc api versions and use

https://github.com/Azure/azure-cli/blob/53c9ce261bd9db21ee02bd4247577deff512d37e/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L155

Reply:
Makes sense - will make this change , after rebasing, since looks like I need to re-record some tests anyway :)